android/src/app/organicmaps/MwmActivity.java

```diff
@@ -1539,9 +1533,18 @@ public void onNavigationStarted()
   {
     closeFloatingToolbarsAndPanels(true);
     ThemeSwitcher.INSTANCE.restart(isMapRendererActive());
-    mNavigationController.start(this);
     mMapButtonsViewModel.setLayoutMode(MapButtonsController.LayoutMode.navigation);
     refreshLightStatusBar();
+
+    // Don't start the background navigation service without fine location.
```

Comment:
Does it make sense to start foreground navigation without the fine location?

Reply:
No, it doesn't make sense. COARSE location returns some random point in the city. It is not usable for voice instructions & navigation.

Reply:
This comment implies that navigation can be started with a coarse location. Does it?

Reply:
Yes, in the current master "START" button can be pressed even without location. This patch doesn't change this behavior. The service is not started without FINE location because it is not really useful.